scipy/stats/tests/test_distributions.py

```diff
@@ -7955,6 +7955,30 @@ def test_trapezoid_vect(self):
 
         assert_allclose(v, res.T.reshape(v.shape), atol=1e-15)
 
+    def test_trapezoid_fit_convergence_gh23503(self):
```

Comment:
`test_continuous` is not well named, but it is just for the new distribution infrastructure. `test_distribution.py` (here) or maybe `test_fit` are more appropriate places for an (old) distribution-specific fit test.